### ISSUE TITLE
Support for Section Headers?

### ISSUE THREAD

**Issue by clintandrewhall** [650Industries/react-native-invertible-scroll-view#8]
Section headers end up looking pretty strange with the transform, (upside down).  I'm looking at the source code and I'm trying to figure out a way to render section headers at the _end_ of a blob section and right-side up.  Any suggestions?

**Comment by ide** [comment]
Not sure why they're appearing upside down, but that should be fixable. We could move around the sticky header children and modify stickyHeaderIndices so that they come at the end of each section, but there's no way to have them dock to the top of the screen if that's what you're wondering.

**Comment by clintandrewhall** [comment]
Nope, just wanting them to appear properly and in the right places.  Thanks for the headstart!

**Comment by PhilippKrone** [comment]
@ide Is there any plan to implement section headers any time soon?

**Comment by ide** [comment]
@PhilippKrone No plans right now. We could fix the inverted section headers pretty easily (send a PR?) but getting them to dock at the top of the screen (ex: like WhatsApp) would require modifications to RCTScrollView itself. This component was written mostly for sectionless chat/comment threads so section headers weren't considered much.

**Comment by alizahid** [comment]
@clintandrewhall Did you ever implement this?
